src/bp-core/compatibility/bp-wp-offload-media-helpers.php

```diff
@@ -349,6 +350,19 @@ public function bb_media_offload_delivery_provider( $delivery_provider ) {
 		return $delivery_provider;
 	}
 
+	/**
+	 * Add extra paramter into localize scripts.
+	 *
+	 * @since [BBVERSION]
+	 *
+	 * @param array $params Localize scripts parameter.
+	 *
+	 * @return array
+	 */
+	public function bb_offload_localize_scripts( $params = array() ) {
+		$params['is_as3cf_active'] = true;
+		return $params;
+	}
```

Comment:
```suggestion
	/**
	 * Add extra parameter into localize scripts.
	 *
	 * @since BuddyBos [BBVERSION]
	 *
	 * @param array $params Localize scripts parameter.
	 *
	 * @return array
	 */
	public function bb_offload_localize_scripts( $params = array() ) {
		if ( bp_is_active( 'activity' ) && bp_is_active( 'media' ) ) {
			$params['is_as3cf_active'] = true;
		}

		return $params;
	}
```